perllib/FixMyStreet/App/Controller/Report.pm

```diff
@@ -725,6 +718,38 @@ sub _nearby_json :Private {
     }
 }
 
+# distance in metres
+#
+# A cobrand may optionally have configured:
+# a) distances for an individual (sub)category
+# b) distances for a group/parent category
+# c) distances by mode
+#
+# NOTE: Distances for category or group only apply for the 'suggestions' mode.
+sub _find_distance {
+    my ($self, $c, $params) = @_;
+
+    my $dist;
+    my $mode = $c->get_param('mode');
+    my $cobrand_distances = $c->cobrand->nearby_distances;
+
+    if ($mode && $cobrand_distances) {
+        my $category = $params->{categories}[0];
+        my $group    = $params->{group};
+
+        $dist
+            = $category && $cobrand_distances->{$category} && $mode eq 'suggestions'
```

Comment:
Having the config like this is okay, though it seems like it'd take up a lot of config file space. Wonder if we could restructure so instead of:
```
nearby_distances:
  oxfordshire:
    suggestions: 800
    'Subcat 1 in Group 1':
      suggestions: 100
    'Group 1':
      suggestions: 400
```

it was structured the other way round, as:
```
nearby_distances:
  oxfordshire:
    suggestions:
      _fallback: 800
      'Subcat 1 in Group 1': 100
      'Group 1': 400
```

(_fallback matching other cobrand features)

Reply:
Would that require restructuring the existing config as e.g. it is for live? I.e.
```
nearby_distances:
    _fallback:
      inspector: 1000
      suggestions: 250
    brent: 
      inspector: 1000
      suggestions: 15
    ...
```
would become
```
nearby_distances:
    _fallback:
      inspector: 1000
      suggestions:
        _fallback: 15
    brent: 
      inspector: 1000
      suggestions:
        _fallback: 15
    ...
```

I'll check if this is the case.

Reply:
Ah didn't realise you'd already replied 😅